### PR TITLE
feat: implement sample_orientation_from_CRC workflow

### PR DESCRIPTION
**Pull request by gcapes**
I'm not sure if the `doc` string in the workflow is still ok, or needs updating?

### CODE REVIEW

File: matflow/data/scripts/mtex/sample_orientations_CRC.m
Comment (aplowman) [inline matflow/data/scripts/mtex/sample_orientations_CRC.m:1]: This seems to work anyway, but for consistency with the other MTEX scripts, can you rename this function to `sample_orientations_CRC` so it matches the file name?

**Review by aplowman — APPROVED** [review]

**Review by aplowman — APPROVED** [review]

**Review by aplowman — CHANGES_REQUESTED** [review]
I forgot, could you also update the README table, to show this has now been implemented? Thanks!

**Comment by gcapes** [comment]
I've updated the README, but need to wait until this is merged to get the link to the docs. I've added a placeholder for now so it doesn't get forgotten.

**Comment by aplowman** [comment]
> I've updated the README, but need to wait until this is merged to get the link to the docs. I've added a placeholder for now so it doesn't get forgotten.

You can predict what the URL will be based on the task schema. It's (objective + method + implementation) joined with hyphens, all lowercase. So in this case it will be:

`https://docs.matflow.io/stable/reference/template_components/task_schemas.html#sample-orientations-from-crc-file-mtex`

It's not the end of the world if we get the prediction wrong, but it's easier to do it here so we don't forget about it. You can also build the docs locally to check what it is.

**Review by aplowman — APPROVED** [review]

**Pull request merged**
